css/utility.css

```diff
@@ -9,3 +9,19 @@
 .d-block {
   display: block;
 }
+
+.appearance {
+  appearance: none;
+}
```

Comment:
Use display none or visibility hidden instead, this style is not as stable as the mentioned ones esp for older browsers.

Reply:
I've tried that ,they make the whole input disappear which isn't what i'm trying to achieve , also i tried using the before and after pseudo class to manipulate it which didn't work eventually ...